cmd/root.go

```diff
@@ -11,22 +11,35 @@ import (
 	"github.com/spf13/viper"
 )
 
-// RootCmd represents the base command when called without any subcommands
-var RootCmd = &cobra.Command{
-	Use:   "pomodoro",
-	Short: "Pomodoro CLI",
-	Long:  "A simple Pomodoro command-line client for the Open Pomodoro format",
-}
-
 var (
 	client   *openpomodoro.Client
-	settings *openpomodoro.Settings
+	settings = &openpomodoro.DefaultSettings
```

Comment:
Since the `settings` var is not initalized during `init()` anymore, we need to load it with the default settings otherwise the `start` command will segfault by trying to access `settings` on init. (see `start.go:31`)